tests/superset_test_config.py

```diff
@@ -50,6 +50,7 @@
 SQL_MAX_ROW = 666
 SQLLAB_CTAS_NO_LIMIT = True  # SQL_MAX_ROW will not take affect for the CTA queries
 FEATURE_FLAGS = {
+    **FEATURE_FLAGS,
```

Comment:
So we can override the configs from `~/.superset/config.py`

Reply:
Does this do anything? `FEATURE_FLAGS` is empty by default in the config

Reply:
But `SUPERSET_CONFIG_PATH` (a config file) is loaded before `SUPERSET_CONFIG` (a Python module). This allows users to override the feature flags in `SUPERSET_CONFIG_PATH` (a file not tracked in Git) when starting superset like this:

```
SUPERSET_CONFIG=tests.superset_test_config flask run -p 8088 --no-debugger
```